qstash/api/queues/get.mdx

```diff
@@ -0,0 +1,85 @@
+---
+title: "Get a Queue"
+description: "Retrieves a queue"
+api: "GET https://qstash.upstash.io/v2/queues/{queueName}"
+authMethod: "bearer"
+---
+
+## Request
+
+<ParamField path="queueName" type="string" required>
+  The name of the queue to retrieve.
+</ParamField>
+
+## Response
+
+<ResponseField name="createdAt" type="int" required>
+  The creation time of the queue. UnixMilli
+</ResponseField>
+<ResponseField name="updatedAt" type="int" required>
+  The update time of the queue. UnixMilli
+</ResponseField>
+<ResponseField name="name" type="string" required>
+  The name of the queue.
+</ResponseField>
+<ResponseField name="parallelism" type="int" required> 
```

Comment:
I think it makes sense to have default as 1 instead of making it required. Thoughts?

Reply:
If you do just send a request to the queue without creating it will be created with parallelism 1. So this upsert will be mostly to change the parallelism to something else. So required looks good for now to me. IF we add a new field in the future, we might remove the required.